sdk/runtime/runenv.go

```diff
@@ -7,28 +7,30 @@ import (
 	"os"
 	"strconv"
 	"strings"
+	"time"
 
 	"go.uber.org/zap"
 
 	"github.com/dustin/go-humanize"
 )
 
 const (
-	EnvTestPlan               = "TEST_PLAN"
 	EnvTestBranch             = "TEST_BRANCH"
 	EnvTestCase               = "TEST_CASE"
-	EnvTestTag                = "TEST_TAG"
-	EnvTestRun                = "TEST_RUN"
-	EnvTestRepo               = "TEST_REPO"
-	EnvTestSubnet             = "TEST_SUBNET"
 	EnvTestCaseSeq            = "TEST_CASE_SEQ"
-	EnvTestSidecar            = "TEST_SIDECAR"
-	EnvTestInstanceCount      = "TEST_INSTANCE_COUNT"
-	EnvTestInstanceRole       = "TEST_INSTANCE_ROLE"
-	EnvTestInstanceParams     = "TEST_INSTANCE_PARAMS"
 	EnvTestGroupID            = "TEST_GROUP_ID"
 	EnvTestGroupInstanceCount = "TEST_GROUP_INSTANCE_COUNT"
+	EnvTestInstanceCount      = "TEST_INSTANCE_COUNT"
```

Comment:
Sorry for editing more lines, I just sorted these lexical order. 

There is one line I added, for TestStartTime.

Similarly for other areas in this file